lab-abigail/test/food-test.js

```diff
@@ -0,0 +1,232 @@
+'use strict';
+
+const server = require('../server');
+const chai = require('chai');
+const expect = chai.expect;
+const http = require('chai-http');
+
+chai.use(http);
+
+describe('HTTP Server module', function(){
+  before(function(done){
```

Comment:
if you're interested in learning how to do a beforeEach/afterEach for individual requests, let me know! it'll probably come in useful when we start with persistence to our databases (locally or through mongoDB). we could also instantiate a mock object to pass through, as well as clean up our database afterwards for each unit test :)